appium/tests/screenobjects/new-message.screen.ts

```diff
@@ -236,7 +246,17 @@ class NewMessageScreen extends BaseScreen {
     await this.showRecipientInputIfNeeded();
     const addedRecipientEl = await $(`~aid-${type}-${order}-label`);
     await ElementHelper.waitAndClick(addedRecipientEl);
-    await driver.sendKeys(['\b']); // backspace
+    await ElementHelper.waitAndClick(await this.recipientPopupRemoveButton);
+    await ElementHelper.waitElementInvisible(addedRecipientEl);
```

Comment:
Both backspace and tap+remove should work, so we should ideally test both. Let's also add back the test for `await driver.sendKeys(['\b']); // backspace` here, in another issue.